.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Check whether minirust succeeds to build.
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: download minirust-tooling
+        run: |
+          git clone "https://github.com/memoryleak47/minirust-tooling" ../minirust-tooling
+          cd ../minirust-tooling
+          git checkout 19f3392 # Fixed commit, we need to bump this occasionally.
+
+      - name: link
+        run: |
+          ln -s ${{github.workspace}} ../minirust-tooling/minirust
+
+      - name: setup nightly
+        run: |
+          rustup install nightly
```

Comment:
We also need to pin the nightly, since bumping the nightly might change/break some features we use.

Reply:
Yeah, thats a good point.
I pinned it in `minirust-tooling` now - wanted to do that anyways.

Reply:
`minirust-tooling/specr-transpile` transpiles into `minirust-tooling/gen-minirust`, which is affected by the `minirust-tooling/rust-toolchain.toml`.

Reply:
Ah, a `rust-toolchain.toml`... I'm not a huge fan of those but I guess it'll do for now.